website/api/blockchain-integration/eth-examples.md

```diff
@@ -6,6 +6,10 @@ While all of the [zkVM examples][zkvm-examples] can be run on Bonsai by [configu
 
 The [RISC Zero Foundry Template][foundry-template] provides a minimal application that can act as a template for developing your application.
 
+### Steel
+
+[Steel] is a library for [view call proofs], which enables users to easily integrate claims about Ethereum state into zkVM applications.
```

Comment:
I've been finding it helpful to use tags like this for links. It helps understand where the link goes when reading the Markdown text... and [my linter](https://github.com/DavidAnson/markdownlint) yells at me when the links break with this format, but not without the tag.
```suggestion
[Steel][steel-repo] is a library for [view call proofs][steel-blog], which enables users to easily integrate claims about Ethereum state into zkVM applications.
```